_docs/never-subclass-components.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Never Subclass Components
+title: Never Subclass Other Components
```

Comment:
@natansh The reason I thought other was okay was the last sentence in that first paragraph actually used the word other.
> Don't subclass other component classes.

But I agree that it is probably better for the content to be changed.